insights/parsers/kpatch_list.py

```diff
@@ -0,0 +1,71 @@
+"""
+KpatchList - command ``/usr/sbin/kpatch list``
+==============================================
+
+The ``/usr/sbin/kpatch list`` command provides information about
+the installed patch modules.
+
+Sample content from command ``/usr/sbin/kpatch list`` is::
+    Loaded patch modules:
+    kpatch_3_10_0_1062_1_1_1_4 [enabled]
+
+    Installed patch modules:
+    kpatch_3_10_0_1062_1_1_1_4 (3.10.0-1062.1.1.el7.x86_64)
+
+Examples:
+    >>> 'kpatch_3_10_0_1062_1_1_1_4' in kpatchs.loaded
+    True
+    >>> kpatchs.loaded.get('kpatch_3_10_0_1062_1_1_1_4')
+    'enabled'
+    >>> 'kpatch_3_10_0_1062_1_1_1_4' in kpatchs.installed
+    True
+    >>> kpatchs.installed.get('kpatch_3_10_0_1062_1_1_1_4')
+    '3.10.0-1062.1.1.el7.x86_64'
+"""
+
+from insights.specs import Specs
+from insights.parsers import SkipException
+from insights import parser, CommandParser
+
+
+@parser(Specs.kpatch_list)
+class KpatchList(CommandParser):
+    """Class for command: /usr/sbin/kpatch list"""
+    def parse_content(self, content):
+        if content is None or len(content) == 0:
+            raise SkipException("No Data from command: /usr/sbin/kpatch list")
+
+        self._loaded = {}
```

Comment:
The loaded can be a single string.

Is it possible that multiple kpatches are loaded at the same time?

Reply:
In this [patch](https://www.redhat.com/archives/kpatch/2014-July/msg00003.html), it says:
"One or more patch modules may be configured to load at boot time ... ..."

Reply:
You'd better add a test case for more than patches are loaded. To make sure the current logic could cover those cases.

Then as per the actual cases, to decide the type of the `loaded` (list or dict), I mean when all the 2nd columns are the same for `loaded` kpatches, it's not necessary to return them.

Reply:
I patch two kmod and get the following ouput:
```
# kpatch list
Loaded patch modules:
livepatch_other_path [enabled]
livepatch_test [enabled]

Installed patch modules:
```